cypress/e2e/hub/my-imports.cy.ts

```diff
@@ -15,34 +15,23 @@ function visitImports(namespace: string) {
   cy.verifyPageTitle('My imports');
 }
 
-describe.skip('My imports', () => {
+describe('My imports', () => {
   const validCollection = {
     namespace: `testnamespace${randomString(4, undefined, { isLowercase: true })}`,
     name: `testcollection_${randomString(4, undefined, { isLowercase: true })}`,
     version: '1.0.0',
   };
 
-  const invalidCollection = {
-    namespace: `testnamespace${randomString(4, undefined, { isLowercase: true })}`,
-    name: `testcollection_INVALID${randomString(4)}`,
-    version: '1.0.0',
-  };
-
   before(() => {
-    cy.createNamespace(validCollection.namespace);
-    cy.galaxykit('-i collection upload', validCollection.namespace, validCollection.name);
-
-    cy.createNamespace(invalidCollection.namespace);
-    cy.galaxykit('-i collection upload', invalidCollection.namespace, invalidCollection.name);
+    cy.createHubNamespace({ namespace: { name: validCollection.namespace } }).then(() => {
+      cy.uploadCollection(validCollection.name, validCollection.namespace, validCollection.version);
```

Comment:
This is a `galaxykit` command, correct?

Reply:
Yes, it is for now. There's [issue](https://issues.redhat.com/browse/AAP-29390) to remove it from this command.